docs/source/user_guide/installation/GNU-linux.rst

```diff
@@ -133,9 +131,29 @@ order to use the wrapped SUNDIALS ODE and DAE
 
    .. code:: bash
 
-	  pip install scikits.odes
+      brew install openblas
+      pybamm_install_odes
 
-   Assuming that SUNDIALS was installed as described :ref:`above<user-install-label>`.
+   The ``pybamm_install_odes`` command, installed with PyBaMM, automatically downloads and installs the SUNDIALS library on your
+   system (under ``~/.local``), before installing ``scikits.odes``. (Alternatively, one can install SUNDIALS without this script and run ``pip install pybamm[odes]`` to install ``pybamm`` with ``scikits.odes``.)
+
+To avoid installation failures when using ``pip install pybamm[odes]``, make sure to set the ``SUNDIALS_INST`` environment variable. If you have installed SUNDIALS using Homebrew, set the variable to the appropriate location. For example:
+
+.. code:: bash
+
+   export SUNDIALS_INST=$(brew --prefix sundials)
+
+Ensure that the path matches the installation location on your system. You can verify the installation location by running:
+
+.. code:: bash
+
+   brew info sundials
+
+Look for the installation path, and use that path to set the ``SUNDIALS_INST`` variable.
+
+Note: The location where Homebrew installs SUNDIALS might vary based on the system architecture (ARM or Intel). Adjust the path in the ``export SUNDIALS_INST`` command accordingly.
```

Comment:
Can we link the `scikits.odes` installation docs here as well (because it requires other things like C compilers and `gfortran` that may not be present on someone's system)?

Reply:
Sure, will do that.

Reply:
It might also make sense to rename this file to `gnu-linux-mac.rst` or something btw and fix any broken links, the heading at the top of the file says

```rst
GNU-Linux & MacOS
=================
```

but the file name is misleading for contributors

Reply:
Sure, thanks for pointing it out.